configure.ac

```diff
@@ -1,10 +1,13 @@
 AC_PREREQ([2.69])
 AC_INIT([iBASIC], [0.1], [ian@mnementh.co.uk], [ibasic], [https://github.com/spyro2/ibasic])
+AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
+AM_SILENT_RULES([yes])
```

Comment:
Nowadays, the compiler command-line is so damned ridiculous, it's kind of pointless to have them scroll by (I remember when that wasn't the case!). With this, it'll default to a "silent mode", that looks a bit like this:

```
  CC     foo_main.o
  CCLD   foo
```

You can specify `V=1` (verbose) on an invocation of `make` to go back to normal just once, or pass `--disable-silent-rules` to the `configure` invocation to make that sticky.